### PR TITLE
orientdb: correct service behavior in dserver

### PR DESCRIPTION
**Pull request by bougyman**

### CODE REVIEW

**Comment by xtraeme** [comment]
If you've tested this, what's stopping you? :+1:

File: srcpkgs/orientdb/patches/00_bash_not_sh.patch [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
++++ ./bin/dserver.sh	2015-03-07 11:32:14.817541145 -0600
+@@ -1,5 +1,5 @@
 -#!/bin/sh
 +#!/bin/bash
  #
  # Copyright (c) Orient Technologies LTD (http://www.orientechnologies.com)
  #
+-${0/%dserver.sh/server.sh} -Ddistributed=true $*
++exec ${0/%dserver.sh/server.sh} -Ddistributed=true $*
+Only in .: that.patch
```
Comment (Gottox) [inline srcpkgs/orientdb/patches/00_bash_not_sh.patch]: This can be removed.

File: srcpkgs/orientdb/patches/00_bash_not_sh.patch [hunk 2]
```diff
@@ -1,9 +1,12 @@
-diff -rupN orientdb-community-2.0.4.orig/bin/dserver.sh orientdb-community-2.0.4/bin/dserver.sh
---- orientdb-community-2.0.4.orig/bin/dserver.sh	2015-03-03 17:24:25.000000000 -0600
-+++ orientdb-community-2.0.4/bin/dserver.sh	2015-03-06 19:21:41.272493588 -0600
-@@ -1,4 +1,4 @@
+diff -ru a/orientdb-community-2.0.4.orig/bin/dserver.sh ./bin/dserver.sh
+--- a/orientdb-community-2.0.4.orig/bin/dserver.sh	2015-03-03 17:24:25.000000000 -0600
++++ ./bin/dserver.sh	2015-03-07 11:32:14.817541145 -0600
+@@ -1,5 +1,5 @@
 -#!/bin/sh
 +#!/bin/bash
```
Comment (Gottox) [inline srcpkgs/orientdb/patches/00_bash_not_sh.patch]: What's the reason to switch to bash?

**Comment by Gottox** [comment]
any news?

File: srcpkgs/orientdb/patches/00_bash_not_sh.patch (hunk 2 above)
Comment (bougyman) [inline srcpkgs/orientdb/patches/00_bash_not_sh.patch]: ${0/%dserver.sh/server.sh} -Ddistributed=true $* is not valid in sh.

**Pull request merged**